Packs/BmcITSM/Integrations/BmcITSM/BmcITSM.yml

```diff
@@ -0,0 +1,2860 @@
+          predefined:
+            - ""
+        - name: assigned_group
+          description: It makes up the third tier of the Problem Assignee’s Support Organization data structure.
+          required: false
+          isArray: false
+          defaultValue: ""
+          predefined:
+            - ""
+        - name: investigation_justification
+          description: The justification for the ticket creation.
+          required: false
+          isArray: false
+          defaultValue: ""
+          predefined:
+            - ""
+        - name: assignee
+          description: The full name of the staff member to whom the ticket will be assigned to. It can be retrieved by using the 'bmc-itsm-user-list' command.
+          required: false
+          isArray: false
+          defaultValue: ""
+          predefined:
+            - ""
+        - name: assignee_pbm_mgr
+          description: The full name of the staff member to whom the ticket will be assign to as the problem coordinator. It can be retrieved by using the 'bmc-itsm-user-list' command.
+          required: false
+          isArray: false
+          defaultValue: ""
+          predefined:
+            - ""
+        - name: temporary_workaround
+          description: Error workaround.
+          required: false
+          isArray: false
+          defaultValue: ""
+          predefined:
+            - ""
+        - name: additional_fields
+          description: 'The fields which are not present in the current argument list can be added here in the format "fieldname1=value;fieldname2=value". Possible fields: Assigned Group, Assignee or any other custom field.'
+          required: false
+          isArray: false
+          defaultValue: ""
+          predefined:
+            - ""
+      outputs:
+        - contextPath: BmcITSM.KnownError.RequestID
+          description: Known Error unique Request ID.
+          type: String
+        - contextPath: BmcITSM.KnownError.DisplayID
+          description: KnownError Display ID.
+          type: String
+        - contextPath: BmcITSM.KnownError.CreateDate
+          description: KnownError Create time in UTC.
+          type: Date
+    - name: bmc-itsm-known-error-update
+      description: "Update Known Error ticket type. "
+      deprecated: false
+      arguments:
+        - name: ticket_request_id
+          description: Known Error request ID.
+          required: true
+          isArray: false
+          defaultValue: ""
+          predefined:
+            - ""
+        - name: status
+          description: Known error status.
+          required: false
+          isArray: false
+          defaultValue: ""
+          auto: PREDEFINED
+          predefined:
+            - Assigned
+            - Scheduled For Correction
+            - Assigned To Vendor
+            - No Action Planned
+            - Corrected
+            - Closed
+            - Cancelled
+        - name: summary
+          description: known error summary.
```

Comment:
here too. For example "known error" is lowercase while its upercase in the argument above